packages/mux-uploader/src/mux-uploader-drop.ts

```diff
@@ -41,23 +66,38 @@ const Attributes = {
 };
 
 class MuxUploaderDropElement extends globalThis.HTMLElement {
-  overlayText: HTMLElement;
+  #overlayTextEl: HTMLElement;
+  #uploaderEl: MuxUploaderElement | null | undefined;
+  #headingEl: HTMLSpanElement | null | undefined;
+
+  #controller: AbortController | undefined;
```

Comment:
Might be worth naming it `#abortController`. Since we also have an `uploaderController`, it would make it easier not to get confused between the two.